frame/alliance/src/lib.rs

```diff
@@ -408,6 +428,8 @@ pub mod pallet {
 		UnscrupulousItemAdded { items: Vec<UnscrupulousItemOf<T, I>> },
 		/// Accounts or websites have been removed from the list of unscrupulous items.
 		UnscrupulousItemRemoved { items: Vec<UnscrupulousItemOf<T, I>> },
+		/// Alliance disband.
+		AllianceDisband { members: Vec<T::AccountId>, proposals: Vec<T::Hash> },
```

Comment:
Events should be past participles.
```suggestion
		/// Alliance disbanded.
		AllianceDisbanded { members: Vec<T::AccountId>, proposals: Vec<T::Hash> },
```